app/routes/og.blog.$slug.tsx

```diff
@@ -0,0 +1,150 @@
+import invariant from "tiny-invariant";
+import getEmojiRegex from "emoji-regex";
+import { type LoaderFunctionArgs } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { isOpenGraphImageRequest, OpenGraphImageData } from "remix-og-image";
+import {
+  BlogAuthor,
+  getBlogPost,
+  getBlogPostListings,
+} from "~/lib/blog.server";
+import { Wordmark } from "~/ui/logo";
+
+interface AuthorWithImage extends BlogAuthor {
+  imgSrc: string;
+}
+
+export async function openGraphImage() {
```

Comment:
This feels like an extremely Remix way of doing this. Looking forward to hearing your thoughts on this.